File: rtcb/jp.go.aist.rtm.rtcbuilder.vbdotnet/resource/ConfigSet/configset2/MyServiceImpl.vb
Public Class MyServiceImpl
    Inherits MarshalByRefObject
    Implements MyService
    Public Sub setGain(ByVal gain As System.Single) Implements MyService.setGain
        Throw New Exception("The method or operation is not implemented.")
    End Sub

    Public Function getGain() As System.Single Implements MyService.getGain
        Throw New Exception("The method or operation is not implemented.")
    End Function

End Class

File: jp.go.aist.rtm.rtcbuilder.csharp/resource/ConfigSet/configset2/MyServiceImpl.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace foo
{
    public class MyServiceImpl : MarshalByRefObject, MyService
    {
        public void setGain(System.Single gain)
        {
            throw new Exception("The method or operation is not implemented.");
        }
        public System.Single getGain()
        {
            throw new Exception("The method or operation is not implemented.");
        }
    }
}
